recipes/wmctrl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "wmctrl" %}
+{% set version = "0.3" %}
+{% set sha256 = "d806f65ac1554366b6e31d29d7be2e8893996c0acbb2824bbf2b1f49cf628a13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - wmctrl
+
+about:
+  home: https://bitbucket.org/antocuni/wmctrl
+  license: BSD
```

Comment:
Sorry must have missed this one. Yeah I'm ok with it. We have done this before in exceptional circumstances, which this seems to fit under. Would be worthwhile to add an issue in the feedstock about this problem with a link to the upstream issue.